### DISCUSSION TITLE
Switch card

### DISCUSSION THREAD

**Discussion by xPhantomNL** [piitaya/lovelace-mushroom#414] [Ideas]
I have a couple of switches (as in, actual power plugs) that control various devices. I would love to use a card similar looking to the light card (at least for the switches that control lights) and was thinking of creating such a card and create a PR for it. 

I've seen the Light Popup Card from DBuit, and they have this for switches and would love to use such a design for a switch card (although, horizontal ofcourse); 
![image](https://user-images.githubusercontent.com/5745036/165906908-f65d422d-ab09-413c-87b4-b4034a3754b5.png)

Before I go ahead and try creating something, I saw in PR #30 that there used to be a switch card that was removed, what is the 'way-to-go' now, regarding switches? If it isn't something that's desired, I would likely not going to put in the effort.

**Comment by piitaya** [comment]
You can use entity card. It works well 🙂
Your screenshot shows a popup, not a card. I prefer to not popup in Mushroom for now but it will be too complex and a more-info popup already exist in Home Assistant.

**Reply by xPhantomNL** [reply]
I understand, it wasn't meant as a popup but as a card, like;
- Copy the current light card
- Replace the "slider" part with the "switch" part of the popup in the screenshot (as in, the white bar that you move right to turn on, left to turn off)

Here's a quick and dirty mockup of the idea;
![image](https://user-images.githubusercontent.com/5745036/165909656-cda39b18-c009-4333-99c5-a1ff0fd23b0b.png)

**Comment by piitaya** [comment]
Yep, can be a good thing to add it but I think we need to refactor cards (by sharing code between card) before adding new because as there is already a lot of card, it's painful to maintain 🙁
But I keep the idea for the future 👍
